randovania/games/prime2/logic_database/Agon Wastes.txt

```diff
@@ -1885,14 +1885,28 @@ Extra - in_dark_aether: True
   > Event - Dark Agon Temple Key Gate
       Dark Agon Key 1 and Dark Agon Key 2 and Dark Agon Key 3
   > Event - Amorbis
-      Morph Ball Bomb and Morph Ball and After Dark Agon Temple Key Gate
+      All of the following:
+          Morph Ball Bomb and Morph Ball and After Dark Agon Temple Key Gate and Normal Damage and Dark World Damage ≥ 30
+          Any of the following:
+              Combat (Intermediate)
+              All of the following:
+                  Normal Damage ≥ 100
```

Comment:
I'd simplify this to say

`Damage 200`
Or
`Combat Beginner and Damage 100`

It reduces clutter and makes trickless require a little more health, which is a good thing  imo (can adjust the values as needed)